pymobiledevice3/remote/bonjour.py

```diff
@@ -0,0 +1,39 @@
+import itertools
+import time
+from socket import AF_INET6, inet_ntop
+from typing import List
+
+from ifaddr import Adapter, get_adapters
+from zeroconf import ServiceBrowser, ServiceListener, Zeroconf
+from zeroconf.const import _TYPE_AAAA
+
+DEFAULT_BONJOUR_TIMEOUT = 1
+
+
+class RemotedListener(ServiceListener):
+    def __init__(self, adapter: Adapter):
+        super().__init__()
+        self.adapter = adapter
+        self.addresses: List[str] = []
+
+    def add_service(self, zc: Zeroconf, type_: str, name: str) -> None:
+        if name == 'ncm._remoted._tcp.local.':
+            service_info = zc.get_service_info(type_, name)
+            records = zc.cache.async_entries_with_name(service_info.server)
```

Comment:
Isn't `entries_with_name` the better API?

Reply:
refactored